app/views/middleware_messaging/new.html.haml

```diff
@@ -0,0 +1 @@
+= render :partial => 'shared/views/ems_common/form'
```

Comment:
New action also seems irrelevant - we are not creating jms queues and topics from miq at the moment (and not as part of this PR)